dev_guide/builds/triggering_builds.adoc

```diff
@@ -436,6 +436,22 @@ strategy:
 This ensures that the triggered build uses the new image that was just pushed to
 the repository, and the build can be re-run any time with the same inputs.
 
+An image change trigger can be paused allowing for multiple changes on referenced
+image stream before a build is started. Also introducing an image change trigger
+on a build config will initiate a build by default. In order to prevent this
+behavior, you can set the paused attribute to true when introducing the image
+change trigger.
```

Comment:
A suggested rewording of the last two sentences:

`You can set the pause attribute to true on a BuildConfig's ImageChangeTrigger to prevent initiating a build by default.`

Reply:
A suggested rewording of gabe's rewording:

```
You can also set the `paused` attribute to true when initially adding an ImageChangeTrigger to a BuildConfig to prevent a build from being immediately triggered.
```